megamek/src/megamek/server/GameManager.java

```diff
@@ -1631,6 +1631,8 @@ public BVCountHelper() {
     }
 
     private void bvReports(boolean checkBlind) {
+        Vector<Report> playerReport = new Vector<>();
```

Comment:
Ideally, we'd use the modern collection classes, at least when we can do so without having to do a grand refactor. This seems like it could work with List<Report> playerReport = new ArrayList<>()